content/vendor/guides/ha-guide.md

```diff
@@ -0,0 +1,411 @@
+
+The next command will test the stability of the cluster when a node is removed. There are a few things you need to keep in mind:
+
+- The Postgres instance in this application, and the application itself are not configured or set up for High Availability. This test is meant to show the resiliancy capabilities of the Kubernetes embedded cluster installed by KURL and configured through Kots.
+
+- As described in the [Kubernetes Documentation](https://kubernetes.io/docs/concepts/architecture/nodes/#condition), there is a default 5 minute timeout duration. This means that there will be 5 minutes between the time the node goes down and that Kubernetes will finally decide that the node is not coming back and will schedule pods on this node to another node.
+
+To simulate a node becoming unresponsive or simply crashed, you can simply stop the VM corresponding to the node. You are likely not to see much change in the pods during the first five minutes, other than some pods erroring out. After 5 minutes or so, you will start to see pods being terminated and scheduled on the remaining nodes. 
+
+In this scenario, the downtime for Postgres was about 6 - 7 minutes. This includes the Kubernetes time out of 5 minutes, 1 minute or less for ekco to detect the node being evicted, and the remainig time is for Postgres to start up on a new node.
+
+
+## Troubleshooting
+
+While the above process has been tested several times and tested succesfully the retention of data, there were a couple of issues that you may run into and are documented below
+
+### Postgres pod will not come up after deleting the pod
+
+In the first test above, after deleting the Postgres pod, the new one would not come up. After looking at the logs of the pod found errors like 
+
+```shell
+find: ‘/var/lib/postgresql/data/pgdata/pg_stat_tmp/global.stat’: Structure needs cleaning
+chown: cannot access '/var/lib/postgresql/data/pgdata/pg_stat_tmp/global.stat': Structure needs cleaning
+```
+This appeared to be more of an OS issue, so I deleted this pod. When the new one was scheduled all was fine.
+
+### Postgres unable to mount to data directory
+
+In some occassions when you shut down the node that Postgres is running on, you may run into a situation where the new pod that gets scheduled on one of the remaining nodes is not able to come up due to volume mount issues. Running `kubectl describe` on the Postgres pod may show an error like the following:
+
+![PostgresDesribe](/images/guides/kots/ha-cluster-postgres-describe.png)
+
+The first thing to check is to ensure you have followed the [configuring ekco] section. Ekco helps with ensuring that any resources that were running on the stopped node are properly deleted and moved to a different node. Without ekco, the claim on the PVC associated to the Postgres pod that was running on the stopped node sometimes doesn't get properly cleared and when the new pod gets scheduled on another node, it is not able to lay a claim to the volume.
```

Comment:
missing link here I think? `[configuring ekco]`

Reply:
This statement about how the Claim is cleared sounds right but I don't know enough about EKCO to say for sure. Let's check w/ core on the phrasing here?

Reply:
Good catch on the link. I ended up changing the whole Troubleshooting section and ended up removing that part.